terraform/modules/data-share-service/codedeploy.tf

```diff
@@ -16,7 +16,7 @@ resource "aws_codedeploy_deployment_group" "gdx_data_share_poc" {
 
   blue_green_deployment_config {
     deployment_ready_option {
-      action_on_timeout = "CONTINUE_DEPLOYMENT"
+      action_on_timeout    = "CONTINUE_DEPLOYMENT"
```

Comment:
Unrelated but made the format checker unhappy

Reply:
yes thanks, I am blind